medical_appointment/data/medical_appointment_data.xml

```diff
@@ -85,4 +85,8 @@
 <p>Your appointment with Dr. ${object.physician_id.lastname} on ${ctx.get('appointment_date') or object.appointment_date[:10]} ${ctx.get('appointment_time') or object.appointment_date[11:]} has been confirmed.</p>
         ]]></field>
     </record>
+    <record forcecreate="True" id="decimal_appointment" model="decimal.precision">
```

Comment:
Essentially all files need to be split by model, the same way we do with actual models. See [contribution docs, file naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming)